src/overlays/actors/ovl_En_Zo/z_en_zo.c

```diff
@@ -60,7 +61,9 @@ static ColliderCylinderInit sCylinderInit = {
     },
     { 18, 64, 0, { 0, 0, 0 } },
 };
+
 static CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, MASS_IMMOVABLE };
+
 static DamageTable sDamageTable = {
```

Comment:
I suppose you could make an enum for this damagetable if you really wanted to finish this actor off, but since they're all zero, there's not a lot of point, I suppose.

Reply:
Additionally, since objects are now on documentation on the animation headers etc can be completed.